standalone/runtime/src/lib.rs

```diff
@@ -634,7 +639,7 @@ parameter_types! {
 
     pub BetterUnsignedThreshold: Perbill = Perbill::from_rational(1u32, 10_000);
 
-    // worker configs
+    // miner configs
```

Comment:
I remember we have a PR removing all the miner/mining keywords.

Reply:
This from upstream https://github.com/paritytech/polkadot-sdk/blob/master/substrate/bin/node/runtime/src/lib.rs#L705